# Basta criar uma classe que herda de Exception
class TaErradoError(Exception):
    pass


def testar():
    raise TaErradoError('TÃ¡ errado!')


try:
    testar()
except TaErradoError as err:
    print(f"ERRO: {err}")
